src/modules/iframe/components/tools/DrawTool.js

```diff
@@ -107,30 +108,39 @@ export class DrawTool extends MvuElement {
 		};
 
 		const icons = this._getIcons(model);
-		const classes = { 'is-enabled': active };
+		const classes = { 'draw-tool__enable': active };
+
+		const getActiveClass = () => {
+			return active ? 'draw-tool__enable' : 'draw-tool__disable';
+		};
 
 		return html`
 			<style>
 				${css}
 			</style>
 			<div class="draw-tool">
-				<div class="draw-tool__content">
+				<div class="draw-tool__content ${getActiveClass()}">
```

Comment:
Should we add a test for the presence of this class?

Reply:
done